.github/workflows/linters.yml

```diff
@@ -46,4 +46,19 @@ jobs:
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
```

Comment:
**Kindly fix this issue:**
- I'm trying to submit the form once that i have filled the inputs and when you click on the button the application it's refreshed and goes at the top of the page.

Reply:
@carlos-ssh I've checked and confirmed that when you press submit it redirects to formspree. Please recheck again